state-chain/pallets/cf-environment/src/lib.rs

```diff
@@ -228,6 +242,21 @@ pub mod pallet {
 	#[pallet::getter(fn solana_api_environment)]
 	pub type SolanaApiEnvironment<T> = StorageValue<_, SolApiEnvironment, ValueQuery>;
 
+	#[pallet::storage]
+	#[pallet::getter(fn solana_open_contract_swap_accounts)]
+	pub type SolanaOpenContractSwapAccounts<T> =
+		StorageValue<_, Vec<ContractSwapAccountAndSender>, ValueQuery>;
```

Comment:
I think it's likely that this will be stored in the election that witnesses these.

Reply:
I think the election pallet would store the new accounts (the ones not stored here stored here) which it will then report. The SC stores the accounts that it already knows about. Depends of course also on how the witnessing is implemented.

Reply:
I had also discussed this with @kylezs so either of you would take that task (PRO-1521). He can chip in for this discussion.